e2e/homepage.spec.ts

```diff
@@ -7,13 +7,29 @@ test('should display homepage in logged out state', async ({ page }) => {
 
 	await homepageLoggedOut.waitForReady();
 
-	await expect(page).toHaveScreenshot({ fullPage: true });
+	await page.waitForLoadState('networkidle');
+
+	await expect(page).toHaveScreenshot({
+		fullPage: true,
+		animations: 'disabled'
+	});
 });
 
 testWithII('should display homepage in logged in state', async ({ page, iiPage }) => {
 	const homepageLoggedIn = new HomepageLoggedIn({ page, iiPage });
 
 	await homepageLoggedIn.waitForReady();
 
-	await expect(page).toHaveScreenshot({ fullPage: true });
+	await page.waitForLoadState('networkidle');
+
+	const numberOfSlides = await homepageLoggedIn.getNumberOfSlides();
+
+	for (let i = 1; i <= numberOfSlides; i++) {
```

Comment:
I did not put it somewhere else because the carousel is only showing on the homepage. I mean yes it does show when it tests for specific modals but activity page for example does not show the carousel.

I don't yet see why we would want to test the carousel seperately.. Since it is part of the "homepage" imo its totally fine to test it there.

Reply:
well, we have quite a lot of other tests that inherits the homepage (receive, settings, activity, explorer, for example). All of those need a stable carousel too, and the only thing that they have in common at start is `waitForReady` (that makes sense)

Reply:
in fact, if you are going to add this in this test, you should add for all of them, now that I think about it...

Reply:
Thats my point, I don't think the carousel is necessary for the other tests. Yes the homepage is there at some point but the only time it really matters is when the carousel is on the screenshot. And that is only in the homepage test I am pretty sure. In my opinion there is no real use if we add it to a test, but its not tested the way it should.

Reply:
the carousel is currently present on:

- homepage
- activity
- explore
- settings

Reply:
My bad, forget my comment, the carousel is only in the homepage, you are right!

However, the fact remains, that we should test it elsewhere and just keep it static in this specific test (i think @DenysKarmazynDFINITY said something similar too). And we put the "set the carousel as static" in the single test of `waitForReady`, or maybe inside `extendWaitForReady`

Reply:
done